integration-tests/itest/deploy_test.go

```diff
@@ -125,6 +126,53 @@ var _ = Describe("Application deployment", func() {
 
 	})
 
+	Context("A deployed streamlet can be scaled", func() {
+		// A function that calculates the streamlet scale based on pod count
+		type scalePodCorrection func(scale int) int
+		var noCorrection scalePodCorrection = func(scale int) int { return scale }
+		var coordinatorCorrection scalePodCorrection = func(scale int) int { return scale - 1 }
+		scaleCheck := func(streamlet string, scaleCorrection scalePodCorrection) {
```

Comment:
Could have done scaleCorrection as an int instead, fill in -1 or 0, no need for func.

Reply:
this is more to give it a clear name that explains why it's there. i.e. to show the intention.

Reply:
you could just call the int variable `scalePodCorrection`? that is a clear enough name, right?